### PR TITLE
Remove acoustic transactional code

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: basket/news/backends/acoustic.py [hunk 1]
```diff
@@ -7,8 +7,6 @@
 from requests import ConnectionError
 from silverpop.api import Silverpop, SilverpopResponseException
 
-from basket import metrics
```
Comment (stevejalim) [inline basket/news/backends/acoustic.py:10]: Where was this used? I don't see it in removed code so do we still need it?

**Review by stevejalim — APPROVED** [review]
r+ with a question

✉️

File: basket/news/backends/acoustic.py (hunk 1 above)
Comment (robhudson) [reply basket/news/backends/acoustic.py:10]: It was used down below on L127. The other code in this file don't use it so it can be removed.

Comment (stevejalim) [reply basket/news/backends/acoustic.py:10]: d'oh serves me right for reviewing on GH Mobile

**Pull request merged**